templates/donor-donation-success.html.twig

```diff
@@ -1,5 +1,40 @@
 {% extends  '_base.html.twig' %}
 
 {% block content %}
-  <p>Thanks for donating, {{ donorFirstName }}!</p>
+  <p>Dear {{ donorFirstName }} {{ donorLastName }},</p>
+  <p>Thank you for supporting {{ charityName }} with your generous gift through the Big Give.<p>
+  <p>
+    Here is your donation information:
+      <ul>
+        <li>Total for {{ charityName }}: <strong>£{{ totalCharityValueAmount }}</strong></li>
+        <li>Donation: <strong>£{{ donationAmount}}</strong></li>
+        <li>Gift Aid: <strong>£{{ giftAidAmountClaimed }}</strong></li>
+        {% if matchedAmount > 0 %}
+            <li>Match Funds: <strong>£{{ matchedAmount }}</strong></li>
+        {% endif %}
+      </ul>
+
+  <p>
+    Donation reference: <strong>{{ transactionId }}</strong><br>
+    Reference on credit/debit card statement: <span>Charity</span>Checkout<span>.co</span>.uk
```

Comment:
This doesn't feel great. How are screenreaders going to report all these separated bits?

I think it would be better to just leave it alone. If there's a concern that a link is distracting in terms of appearance, one solution would be to make it an actual, deliberate link but style it to look like normal text. It can/should still go to the domain so that donors who want to click it can.